Building DAG of jobs...
Using shell: /bin/bash
Provided cores: 3
Rules claiming more threads will be scaled down.
Job counts:
	count	jobs
	1	all
	4	macs_narrow
	5

rule macs_narrow:
    input: results/bwa/FT_SAMN03496012/SRR1983927-H3K4me1_2.sorted.rmdup.bam
    output: results/macs2/FT_SAMN03496012/SRR1983927-H3K4me1_2_peaks.narrowPeak
    jobid: 23
    wildcards: sample=SRR1983927-H3K4me1_2


rule macs_narrow:
    input: results/bwa/FT_SAMN03496012/SRR1983935-H3K4me1_2.sorted.rmdup.bam
    output: results/macs2/FT_SAMN03496012/SRR1983935-H3K4me1_2_peaks.narrowPeak
    jobid: 29
    wildcards: sample=SRR1983935-H3K4me1_2


rule macs_narrow:
    input: results/bwa/FT_SAMN03496012/SRR1983934-H3K4me1_1.sorted.rmdup.bam
    output: results/macs2/FT_SAMN03496012/SRR1983934-H3K4me1_1_peaks.narrowPeak
    jobid: 11
    wildcards: sample=SRR1983934-H3K4me1_1

Error in rule macs_narrow:
Error in rule macs_narrow:
    jobid: 29
    jobid: 11
Error in rule macs_narrow:
    output: results/macs2/FT_SAMN03496012/SRR1983935-H3K4me1_2_peaks.narrowPeak
    output: results/macs2/FT_SAMN03496012/SRR1983934-H3K4me1_1_peaks.narrowPeak
    jobid: 23


    output: results/macs2/FT_SAMN03496012/SRR1983927-H3K4me1_2_peaks.narrowPeak

RuleException:
CalledProcessError in line 172 of /share/ScratchGeneral/jamtor/projects/hgsoc_repeats/histone-ChIP-seq/Snakefile:
Command ' set -euo pipefail;  /home/jamtor/local/lib/MACS/bin/macs2 callpeak -t results/bwa/FT_SAMN03496012/SRR1983935-H3K4me1_2.sorted.rmdup.bam -c SRR1983929-input_1 --name "HGSOC_SAMN01761041_SRR1983935-H3K4me1_2" --outdir results/macs2/FT_SAMN03496012/ ' returned non-zero exit status 1.
  File "/share/ScratchGeneral/jamtor/projects/hgsoc_repeats/histone-ChIP-seq/Snakefile", line 172, in __rule_macs_narrow
  File "/home/jamtor/local/bin/miniconda2/envs/snakemake/lib/python3.6/concurrent/futures/thread.py", line 56, in run
RuleException:
CalledProcessError in line 172 of /share/ScratchGeneral/jamtor/projects/hgsoc_repeats/histone-ChIP-seq/Snakefile:
Command ' set -euo pipefail;  /home/jamtor/local/lib/MACS/bin/macs2 callpeak -t results/bwa/FT_SAMN03496012/SRR1983934-H3K4me1_1.sorted.rmdup.bam -c SRR1983929-input_1 --name "HGSOC_SAMN01761041_SRR1983934-H3K4me1_1" --outdir results/macs2/FT_SAMN03496012/ ' returned non-zero exit status 1.
  File "/share/ScratchGeneral/jamtor/projects/hgsoc_repeats/histone-ChIP-seq/Snakefile", line 172, in __rule_macs_narrow
  File "/home/jamtor/local/bin/miniconda2/envs/snakemake/lib/python3.6/concurrent/futures/thread.py", line 56, in run
RuleException:
CalledProcessError in line 172 of /share/ScratchGeneral/jamtor/projects/hgsoc_repeats/histone-ChIP-seq/Snakefile:
Command ' set -euo pipefail;  /home/jamtor/local/lib/MACS/bin/macs2 callpeak -t results/bwa/FT_SAMN03496012/SRR1983927-H3K4me1_2.sorted.rmdup.bam -c SRR1983929-input_1 --name "HGSOC_SAMN01761041_SRR1983927-H3K4me1_2" --outdir results/macs2/FT_SAMN03496012/ ' returned non-zero exit status 1.
  File "/share/ScratchGeneral/jamtor/projects/hgsoc_repeats/histone-ChIP-seq/Snakefile", line 172, in __rule_macs_narrow
  File "/home/jamtor/local/bin/miniconda2/envs/snakemake/lib/python3.6/concurrent/futures/thread.py", line 56, in run
Will exit after finishing currently running jobs.
Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
Complete log: /share/ScratchGeneral/jamtor/projects/hgsoc_repeats/histone-ChIP-seq/.snakemake/log/2018-04-27T082350.029134.snakemake.log
